frontend/packages/dev-console/src/components/import/DeployImage.tsx

```diff
@@ -39,6 +37,12 @@ const DeployImage: React.FC<Props> = ({ namespace, projects, activeApplication }
     },
     name: '',
     searchTerm: '',
+    registry: 'external',
+    imageStream: {
+      image: '',
+      tag: '',
+      namespace: '',
+    },
```

Comment:
Why did you need to add this instead of using the `isi` or "image stream image"?

Reply:
I am using these three fields(dropdowns) in UI and using these values an API call is made, which will set `isi` and `image` fields.